config_parser.cc

```diff
@@ -8,6 +8,7 @@
 //   http://lxr.nginx.org/source/src/core/ngx_conf_file.c
 
 #include <cstdio>
+#include <cstdlib>
```

Comment:
It's used for `std::abs()` [here](https://github.com/KyCodeHuynh/nginx-configparser/pull/1#discussion-diff-96321723R223).